esp-config/src/generate/value.rs

```diff
@@ -118,3 +182,22 @@ impl From<String> for Value {
         Value::String(value)
     }
 }
+
+#[cfg(test)]
+mod test {
+    use super::*;
+
+    #[test]
+    fn deserialization_number() {
```

Comment:
Would negative values still work here? I think so as `str::parse` should handle that.